### ISSUE TITLE
how to use EMDLoss

### ISSUE THREAD

**Issue by wmrenr** [yym064/EMDLoss_PyTorch_cpp_extension#1]
我在跑一个点云配准代码，他使用了EMDLoss，如下
from emd import EMDLoss
emd_loss = torch.mean(EMDLoss()(template, source))/(template.size()[1])

但是在他的repo里无法import EMDLoss，于是我找到了你的这个EMDLoss_PyTorch模块并成功下载编译了
请问如何结合你的EMDLoss到他的代码里呢？

**Comment by yym064** [comment]
There is a toy example in loss.py file. please refer this file.
